pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -234,6 +236,20 @@ func (asw *actualStateOfWorld) MarkVolumeAsDetached(
 	asw.DeleteVolumeNode(volumeName, nodeName)
 }
 
+func (asw *actualStateOfWorld) MarkDesireToDetachVolume(
```

Comment:
These names are now misleading. Let's change `MarkDesireToDetachVolume` to
`AddVolumeToReportAsAttached`.